src/uucore/src/lib/features/perms.rs

```diff
@@ -586,3 +645,73 @@ pub fn chown_base(
     };
     executor.exec()
 }
+
+#[cfg(test)]
+mod tests {
+    // Note this useful idiom: importing names from outer (for mod tests) scope.
```

Comment:
Yeah, we know :)

Reply:
Sorry, I copy-pasted it from somewhere else. I'll delete it :)